### PR TITLE
Move model evaluation notebooks from community to official

### PR DESCRIPTION
**Pull request by soheilazangeneh**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.
<br>
Two notebooks added to demonstrate new evaluation components for AutoML regression and classification models.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1433 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Classification -&gt; classification</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>combine these 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Combine these two text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Combine these 2 text cells. </p><p> </p><p>Remove 'Skip this step.'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Add heading</p><p> </p><p>For more details, please go through this <a href="https://cloud.google.com/python/docs/reference/aiplatform/latest/google.cloud.aiplatform.AutoMLTabularTrainingJob" target="_blank">documentation for <strong>AutoMLTabularTrainingJob</strong> Class</a>. -&gt;</p><p>Learn more about [Class AutoMLTabularTrainingJob](http://....)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Add heading.</p><p> </p><p>Model -&gt; Vertex AI Model</p><p> </p><p>Ditto on link style:</p><p> </p><p>Learn more about [title](link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>Vertex XAI&nbsp; -&gt; Vertex Explainable AI</p><p>AutoML Tables -&gt; AutoML Tabular</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>use</p><p> </p><p>Learn more about [title](link).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

File: notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb [hunk 2]
```diff
@@ -0,0 +1,1485 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>Regression -&gt; regression</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>AutoML regression model -&gt; AutoML Tabular regression model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>combine 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>combine 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>combine 2 text cells.</p><p>Remove Skip this step.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>Add heading</p><p> </p><p>Model -&gt; AutoML Model</p><p> </p><p>GCS -&gt; Cloud Storage</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>combine with above text cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (andrewferlitsch) [inline notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>make it a habit to remove the trailing comma on the last parameter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (sudarshan-SpringML) [reply notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>done</p>

File: notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb (hunk 1 above)
Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>updated.</p>

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>updated.</p>

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>done.</p>

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>added.</p>

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>updated.</p>

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>done.</p>

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>done.</p>

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>done.</p>

Comment (krishr2d2) [reply notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb:1]: <p>updated.</p>

File: notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb (hunk 2 above)
Comment (sudarshan-SpringML) [reply notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>done</p>

Comment (sudarshan-SpringML) [reply notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>done</p>

Comment (sudarshan-SpringML) [reply notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>done</p>

Comment (sudarshan-SpringML) [reply notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb:1]: <p>I removed trailing comma on the last parameter. But when I run linter test, comma is added automatically</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**